### DISCUSSION TITLE
help with Amsamotion CPU224XP (S7 200 compatible)

### DISCUSSION THREAD

**Discussion by corerun** [gijzelaerr/python-snap7#288] [Q&A] [answered]
Hi I'm a student, I'm a newbie with Python and PLC, I try to connect an S7 200 CPU224XP with a Raspberry Pi and I got the connection working, but I don't understand how can I simple enable Q0.0.

I got this https://imgur.com/a/GVfYR2p

Can anyone help me? I only want enable I0.x and Q0.x, thank you in advance.

I built this panel for testing programs https://imgur.com/a/n7ZL6PY

**Comment by lautarodapin** [comment, marked as answer]
Try using the logo client, is not well documented but should be easier to use for this cases

**Reply by corerun** [reply]
Where I can find it? Do you have a link please?

Thank you.

**Reply by lautarodapin** [reply]
https://python-snap7.readthedocs.io/en/1.0/logo.html
These are the docs
Reads from VM addresses of Siemens Logo. Examples: read(“V40”) / read(“VW64”) / read(“V10.2”)

Parameters:	vm_address – of Logo memory (e.g. V30.1, VW32, V24)
Returns:	integer
